src/control-service.vala

```diff
@@ -79,6 +79,10 @@ namespace Frida {
 		construct {
 			host_session.spawn_added.connect (notify_spawn_added);
 			host_session.agent_session_detached.connect (on_agent_session_detached);
+			host_session.child_added.connect (notify_child_added);
+			host_session.child_removed.connect (notify_child_removed);
+			host_session.process_crashed.connect (notify_process_crashed);
+			host_session.output.connect (notify_output);
```

Comment:
- Would be good to add these between `spawn_added` and `agent_session_detached`, so they follow the same order as in the `HostSession` interface.
- While at it we should also wire up `uninjected`.